### PR TITLE
Fix: Parquet schema that is readable by microsoft ADF and Purview

### PR DESCRIPTION
**Pull request by chadnaparite**
## Description

Work Item ID: https://dev.azure.com/CluedIn-io/CluedIn/_workitems/edit/40327

## How has it been tested? 
Locally with Docker
- adlg2 account: adlsg2testingapac
- purview account: cluedin

### CODE REVIEW

File: src/Connector.AzureAIStudio/Connector/AzureAIStudioExportEntitiesJob.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public AzureAIStudioExportEntitiesJob(
     {
     }
 
-    protected override string GetOutputFileName(Guid streamId, DateTime asOfTime, string outputFormat)
+    protected override string GetOutputFileName(IDataLakeJobData configuration, Guid streamId, string containerName, DateTime asOfTime, string outputFormat)
```
Comment (hopengfoong) [inline src/Connector.AzureAIStudio/Connector/AzureAIStudioExportEntitiesJob.cs:21]: I think same thing should be done for DataBricks & Synapse Data Engineering too?

File: docs/guides/FilenamePatterns.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  * formatString accepts formats available to C# Guid.ToString() method. Please refer to [here](https://learn.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=net-8.0) for more information.
+  * When using {StreamId} without formatString, it defaults to "D" format. Example values: `ba4afc12-f6dc-4394-b9d5-68f6dacf3b3b`.
+* {OutputFormat} and {OutputFormat:formatString}
+  * formatString accepts the following:-
+    * ToUpperInvariant
+    * ToUpper (equivalent to ToUpperInvariant)
+    * ToLower
+    * ToUpperInvariant (equivalent to ToLowerInvariant)
+  * When using {OutputFormat} without formatString, no extra formatting is performed. Example values: `csv`, `parquet`, `json`.
+
```
Comment (hopengfoong) [inline docs/guides/FilenamePatterns.md:18]: Add `ContainerName` too?

Comment (chadnaparite) [reply docs/guides/FilenamePatterns.md:18]: Done adding.

**Review by andrew-hardy — APPROVED** [review]

**Pull request merged**